content/en/notebooks/_index.md

```diff
@@ -19,106 +19,82 @@ further_reading:
+4. [Configure cells](#cell-configuration).
 
-Changes made to a notebook appear automatically, without the need to refresh.
+## Collaboration
 
-Everyone on your team can open or edit any notebook, but a notebook can only be deleted by the creator or by an administrator.
+{{< img src="notebooks/collaboration.png" alt="Indicators of users viewing the notebook and making live edits" style="width:100%;">}}
 
-## Commenting
+Notebooks support real-time collaboration. Presence indicators show who is viewing your notebook at any time and show real-time edits and comments. 
+
+Changes made to a notebook appear automatically, without the need to refresh.
 
-To add a comment, select some text or hover over a graph. The **Add comment** button is displayed to the right of the cell.
+Everyone on your team can open or edit any notebook, but a notebook can only be deleted by the creator or by an administrator.
 
-{{< img src="notebooks/add_comment.png" alt="add a comment to text" style="width:90%;">}}
+### Commenting
 
-To edit or delete a comment you wrote, click the menu on the top-right of your comment.
+To add a comment, select some text or hover over a graph. The **Add comment** icon is displayed to the right of the cell.
 
-{{< img src="notebooks/edit_comments.png" alt="edit comment in notebook" style="width:90%;">}}
+Click the menu on the top-right of your comment to edit or delete a comment you wrote.
 
 View or re-open resolved comments in the Comment History, available in the Notebook cog menu.
 
-{{< img src="notebooks/check_comment_history.png" alt="edit comment in notebook" style="width:90%;">}}
-
 Notebook authors receive email notifications for new comments on their notebooks, and commenters are notified for replies to their comments. Manage your notification settings through `Notifications` in the notebook cog menu.
 
+### Read mode
 
-## Notebook List
-
-The [Notebook List][1] allows you to view and search previously created notebooks. Each notebook's name, creator, and last modified date are displayed. Notebooks are grouped by:
-
-* **My Notebooks**: Notebooks created by you.
-* **Other Notebooks**: Notebooks created by other members of your team.
-
-The Notebook List also displays preset templates and tutorials. Edit these templates directly, and click **Clone** to save your changes.
-
-{{< img src="notebooks/templates.png" alt="templates on the notebook list page" style="width:80%;">}}
-
-## New Notebook
-
-Create a [new notebook][2] from the main navigation: *Notebooks > New Notebook*.
+{{< img src="notebooks/read_mode.png" alt="Read mode drop down menu" style="width:100%;">}}
 
-A new notebook is not saved by default. You must click the **Save** button.
+From the Notebook List, click on the eye icon to open a notebook in View Mode. This allows you to explore the information without edits. You can switch between modes from within the Notebook by selecting the dropdown at the top right of your notebook.
```

Comment:
```suggestion
You can switch between modes from within the Notebook by selecting the dropdown at the top right of your notebook.
```
[Ignorable suggestion] This information about opening a notebook in View Mode from the list page is repeated in the `Notebook List` section. I'm not sure it adds much value in this section since it has no picture of the button, so it's unclear to an unfamiliar reader what we are referencing. 

Also this is a relatively unpopular feature. I don't believe it warrants being mentioned twice in the docs and I think it fits better in the `Notebook List` section since it has a screenshot of the button there